packit_service/models.py

```diff
@@ -440,6 +442,27 @@ def get_all(cls) -> Optional[Iterable["CoprBuildModel"]]:
         with get_sa_session() as session:
             return session.query(CoprBuildModel).order_by(desc(CoprBuildModel.id)).all()
 
+    @classmethod
+    def get_merged_chroots(
+        cls, first: int, last: int
+    ) -> Optional[Iterable["CoprBuildModel"]]:
+        with get_sa_session() as session:
+            builds_list = (
+                session.query(
+                    # Merged row will keep the id of any one row
```

Comment:
Please make a note here, that this is needed b/c `CoprBuildModel.build_id` is a string and we cannot order the way we want based on that.
(Though checking reveals, that Copr V3 API returns build_id as an integer, so not sure, why it's not saved as such... maybe @TomasTomecek recalls?)

Reply:
Added a comment there.

Since we are merging rows here, we need to tell SQL about which column (out of three or four ie the number of targets) we want to keep.

We want the id in our query result to keep have the id of any one row, so we order them by the MIN function. Earlier this week, when where we were trying to get every single thing in one query, we used the MIN function to even get identical stuff like
```

# since theyre identical, just get any one web_url and call the new query column as web_url 
func.min(CoprBuildModel.web_url).label("web_url"),

```
MIN and MAX, for some reason also works on strings, even identical ones.


But now that we had to offload build details to a seperate query, we just dont need to do that.